### DISCUSSION TITLE
Line break syntax

### DISCUSSION THREAD

**Discussion by gargajit** [fastn-stack/fastn#629] [Ideas & RFCs]
FTD syntax similar to HTML tag <br> should be introduced.

Asked by: @saurabh-lohiya @gargajit

**Comment by amitu** [comment]
@saurabh-lohiya, what did you mean by this?

**Comment by saurabh-lohiya** [comment]
I mean like a `<br>` tag in html, we should have some alternative in ftd.

**Reply by amitu** [reply]
Not sure. BR in HTML is considered bad practice, and spacing/margin css properties should be used instead. Can you give me a use case?
